.github/workflows/runtimes-matrix.json

```diff
@@ -28,15 +28,17 @@
     "path": "system-parachains/asset-hubs/asset-hub-kusama",
     "uri": "wss://kusama-asset-hub-rpc.polkadot.io:443",
     "is_relay": false,
-    "blocktime": 12000
+    "blocktime": 12000,
+    "runtime_args": "--overwrite-state-version 1"
```

Comment:
Why do we need this? The on chain version is still `0`?

Reply:
The externalities initialize with the state version of the snapshot (0) and then at the end of the block it is 1 and fails to match the state root.

Reply:
Are we building a block because of the possible multi block migrations?